include/libdnf/repo/repo.hpp

```diff
@@ -374,6 +374,7 @@ class Repo {
     void make_solv_repo();
 
     void load_available_repo(libdnf::repo::LoadFlags flags);
+    void load_repo_modules();
```

Comment:
Do we want to split `load_available_repo` and `load_repo_modules`? Is there any advantage for that approach?

Reply:
I changed it.